demo/installer/grub-arch/install.sh

```diff
@@ -296,6 +298,9 @@ demo_install_uefi_grub()
 
 eval $create_demo_partition $blk_dev
 demo_dev=$(echo $blk_dev | sed -e 's/\(mmcblk[0-9]\)/\1p/')$demo_part
+echo $blk_dev | grep -q nvme && {
```

Comment:
have you tried this on a non-NVME box?

I think the above test should use `if .... ; then ; fi`.  Without it the failure of the `grep` will trigger an overall script failure, as the script is run with `set -e`.